ui/app/(prowler)/compliance/page.tsx

```diff
@@ -36,34 +35,41 @@ export default async function Compliance({
       "filter[state]": "completed",
     },
     pageSize: 50,
+    fields: {
+      scans: "name,completed_at,provider",
+    },
+    include: "provider",
   });
 
   if (!scansData?.data) {
     return <NoScansAvailable />;
   }
 
-  // Expand scans with provider information - only include scans with valid provider
-  const expandedScansData: ExpandedScanData[] = await Promise.all(
-    scansData.data
-      .filter((scan: ScanProps) => scan.relationships?.provider?.data?.id)
-      .map(async (scan: ScanProps) => {
-        const providerId = scan.relationships!.provider!.data!.id;
-
-        const formData = new FormData();
-        formData.append("id", providerId);
-
-        const providerData = await getProvider(formData);
-
-        return {
-          ...scan,
-          providerInfo: {
-            provider: providerData.data.attributes.provider,
-            uid: providerData.data.attributes.uid,
-            alias: providerData.data.attributes.alias,
-          },
-        };
-      }),
-  );
+  // Process scans with provider information from included data
+  const expandedScansData: ExpandedScanData[] = scansData.data
```

Comment:
![image](https://github.com/user-attachments/assets/aba8656b-b339-4789-9e53-bc2602177285)

Reply:
The issue can be `  const type = compliancesData?.data?.[0]?.type; ` it may be `  const type = compliancesData?.data?.type;`

I need to check it